src/Contexts/Adaptors/SubstrateDestinationAdaptor.tsx

```diff
@@ -213,8 +213,14 @@ export const SubstrateDestinationAdaptorProvider = ({
   }, [transactionStatus]);
 
   useEffect(() => {
-    if (transactionStatus === "In Transit" && api && !fallback?.started())
+    if (
```

Comment:
Let's move this complex condition to a variable with a good name and use it inside the if

Reply:
Fixed